### DISCUSSION TITLE
How to Disable BBS ECHO ON Using only AT Command (not using AT+CONFIG)

### DISCUSSION THREAD

**Discussion by shawnmerrick** [bozimmerman/Zimodem#125] [Q&A]
**I'm modifying my BBS software and seeking an AT command to disable BBS ECHO ON without navigating the AT+CONFIG menus.**  (This is not the same as **E0/E1** keystroke echo for command mode.)  I saw the advice to use **"ATR0E0S0=1S41=1A6400"** but this doesn't work IF THE BBS SYSOP HAS ALREADY used **AT+CONFIG** to **ENABLE** the BBS Listener, which defaults to **ECHO ON.**  Our modems are a branch of firmware 3.6.4.  I've also used ATI5 to view all possible register settings looking for any difference when BBS ECHO ON was _not_ enabled but couldn't find anything.  Thanks in advance for any helpful advice.

**Comment by SalvadorCastello** [comment]
Hi!
I have the same problem with all BBS, did you find a solution?
Thanks

**Reply by shawnmerrick** [reply]
Unfortunately, there were no replies to my concern described above.  I do not know if anyone else has resolved it.

**Reply by SalvadorCastello** [reply]
I downloaded WifiModem and it has the same problem but I changed the code to treat "local-echo" when using TELNET.
I used register S65 , when = 1 , it doesn't echo the input.
I will give a closer look on Zimodem to see if I can do the same and I'll let you know.

**Reply by SalvadorCastello** [reply]
Hi I found an "ugly" solution:
Add the following lines after line 110 of the stream.ino program

    if (isTelnet()) {
       if (int(c) > 19)
          Serial.print(char(8)); Serial.print(' '); Serial.print(char(8));
       if (int(c) == 8)   
          Serial.print(" "); 
    } 

When you set register 62 (telnet) this is activated and will avoid LOCALECHO.

It's working fine for me.
Regards,
